batch/2017-01-01.4.0/examples/NodeGet_SelectClause.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "poolId": "poolId",
+    "nodeId": "tvm-1695681911_1-20161122t193202z",
+    "api-version": "2017-01-01.4.0",
+    "client-request-id": "00000000-0000-0000-0000-000000000000",
+    "ocp-data": "Fri, 17 Feb 2017 00:00:00 GMT",
+    "$select": "id%2Cstate"
```

Comment:
I would remove it.  If showing how to use $select adds value, then we should do it on all APIs, as real readers will be looking only at the API they are interested in.  And that sounds like a poor use of our time...!